addons/web/static/src/scss/ui.scss

```diff
@@ -183,3 +183,14 @@ span.o_force_ltr {
         }
     }
 }
+
+.o_has_ui_app_icon {
+    .o_ui_app_icon {
+        width: var(--oi-font-size, 1em);
+        aspect-ratio: 1;
```

Comment:
Still necessary? I don't see any difference without it 🤔

Reply:
Indeed, since app icons are square, this is not necessary. I don't understand why we added this property before on `o_knowledge_icon_search` and `o_add_to_board` svgs 🤔. Any idea ?
Anyway, I'll to remove it. 👍